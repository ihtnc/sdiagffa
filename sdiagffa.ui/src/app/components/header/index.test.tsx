import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import Header from '.'

describe('Header component', () => {
  it('should match snapshot', () => {
    const component = render(<Header />);
    expect(component).toMatchSnapshot();
  })
});