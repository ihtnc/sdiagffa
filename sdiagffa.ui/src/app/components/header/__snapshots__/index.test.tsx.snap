// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header component > should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="text-swblue max-w-lg swcrawl-container"
      >
        <div
          class="flex flex-col gap-4 swcrawl-text"
        >
          <p
            class="font-semibold"
          >
            A long time ago in a galaxy far, far away....
          </p>
          <p>
            The Republic has conducted a census.
          </p>
          <p>
            Details of each character, their name, homeworld, and other pieces of information have been recorded.
          </p>
          <p>
            An officer is trying to identify how characters in the galaxy are related to each other....
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="text-swblue max-w-lg swcrawl-container"
    >
      <div
        class="flex flex-col gap-4 swcrawl-text"
      >
        <p
          class="font-semibold"
        >
          A long time ago in a galaxy far, far away....
        </p>
        <p>
          The Republic has conducted a census.
        </p>
        <p>
          Details of each character, their name, homeworld, and other pieces of information have been recorded.
        </p>
        <p>
          An officer is trying to identify how characters in the galaxy are related to each other....
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
